src/prefect/transactions.py

```diff
@@ -198,6 +199,22 @@ def commit(self) -> bool:
             self.rollback()
             return False
 
+    def run_hook(self, hook, hook_type: str) -> None:
```

Comment:
nit:

was it intentional to make hook_type str as a opposed to the more explicit Literal?

Reply:
Yup it was - I think type hints are about programmatic correctness, and this function/method does in fact accept any string value for the hook type. 

The fact that it happens to be called with only two values isn't something the method needs to enforce or care about, that's more of a byproduct of the business logic.